### PR TITLE
Add comment about need to restart Kolibri after enabling plugins

### PR DESCRIPTION
**Pull request by ivanistheone**
### Summary

A comment added to docs based on suggestion in https://github.com/learningequality/kolibri/issues/6815

### CODE REVIEW

File: docs/backend_architecture/plugins.rst [hunk 1]
```diff
@@ -35,8 +35,10 @@ path for the plugin is ``kolibri_example_plugin`` then it could be enabled by::
 
     kolibri plugin enable kolibri_example_plugin
 
-The above command can be passed multiple plugin names to enable at once. Similarly, to disable the plugin the following
-command can be used::
+The above command can be passed multiple plugin names to enable at once.
```
Comment (indirectlylit) [inline docs/backend_architecture/plugins.rst:38]: Wow, I didn't even know this!

File: docs/backend_architecture/plugins.rst [hunk 2]
```diff
@@ -35,8 +35,10 @@ path for the plugin is ``kolibri_example_plugin`` then it could be enabled by::
...
+If Kolibri is running, it needs to be restarted for the change to take effect.
+
+Similarly, to disable the plugin the following command can be used::
```
Comment (indirectlylit) [inline docs/backend_architecture/plugins.rst]:
```suggestion
Similarly, to disable the plugin the following command can be used:
```

**Review by indirectlylit — APPROVED** [review]
looks good, pending removal of an extraneous colon::

**Comment by ivanistheone** [comment]
I just realized the formatting for bash blocks on this page is broken:
https://kolibri-dev.readthedocs.io/en/develop/backend_architecture/plugins.html

so adding some more fixes to this PR in 5 mins

**Comment by ivanistheone** [comment]
I also replaced the command line example from `kolibri_example_plugin` to more explicit `kolibri.plugins.example_plugin` since most likely use case is to enable/disable one of the internal plugins rather than implement a completely new plugin from scratch.

**Comment by indirectlylit** [comment]
thanks!

**Comment by indirectlylit** [comment]
thanks!

**Pull request merged**